docs/LTS-strategy.md

```diff
@@ -0,0 +1,38 @@
+# Express LTS Strategy
+
+## Goals
+
+* Maintain strict Node.js version support within a single Express major release.
+* Provide good ergonomics and developer experience for Express maintainers
+* Provide good experience for users who want to keep their Express projects that up-to-date
+* Avoid stagnation and blocking innovation that is good for the end users
+
+## Non-Goals
+
+* Only support the active LTS releases of Node.js
+* Support the widest possible set of Node.js versions. We aim to maintain a healthy balance between backwards compatibility and toolchain modernization.
+* Provide timed cadence of Node.js version support dismissal
+
+### Rationale behind dropping support for old Node.js versions
+
+* Avoid blocking Node.js core from making progress and removing deprecated parts of the codebase
+* Avoid getting outdated bug reports or security issues
+* Keep options for adopting newer patterns and tooling open
+
+### Rationale behind conservative policy of breaking changes
+
+* Minimize disruption for end users
+* Be mindful of the fact that some users are still running obsolete versions of Node in production
+
+### Guidelines for dropping support of Node.js versions
```

Comment:
I think this is the section I want us to discuss more deeply. Ideally we also put together a calendar view like Node does to see in comparison to Node majors how express majors would line up.

Reply:
Yeah! Having a calendar is one of the best ways to explain the rules for the end users, but currently is very solid 🙂

Reply:
I think we should make a diagram like node has, maybe even overlay our support *over* the existing node support schedules?

Reply:
@wesleytodd @UlisesGascon I think @ljharb strongly objected to having a timed deprecation without a practical justification. We need to have this established unambiguously within the strategy. Do we or not do time schedule-based Node version deprecation?

Reply:
Can we merge without a consensus on this? I am leaning yes? And then we can open targeted PRs to update these and discuss after?

Reply:
If it’s worded as something like “if a release is warranted, this is the schedule when it will happen - but if not, no release will happen” then my concern is gone.

Reply:
Either way, but it seems easy enough to do it before merging ¯\\\_(ツ)_/¯

Reply:
@kibertoad if you get to it, awesome. If not, lets try and merge this before EOD so we can start those other PRs to clarify and expand.

Reply:
addressed. unless there are major outstanding objections to some of the wording, I would suggest merging it as-is and iterate in follow-up PRs.

@wesleytodd Can I get push permissions for the repo?

Reply:
thanks!